app/scripts/controllers/metametrics-controller.test.ts

```diff
@@ -73,7 +73,6 @@ const DEFAULT_TEST_CONTEXT = {
   app: {
     name: 'MetaMask Extension',
     version: VERSION,
-    extensionId: MOCK_EXTENSION_ID,
```

Comment:
This ID is only set on MMI builds. It was present in the unit tests because the unit tests don't do any code fence processing, so they end up running on a version of the code-under-test with all fenced code enabled.